.travis.yml

```diff
@@ -25,6 +25,7 @@ addons:
   postgresql: '9.5'
 before_install:
 - source ${TRAVIS_BUILD_DIR}/tools/ci/before_install.sh
+- mkdir ${TRAVIS_BUILD_DIR}/log
```

Comment:
Should this be done before `before_install.sh`?

Reply:
this works, but I'll change - thnx